Apps/APN/ios/AppDelegate.h

```diff
@@ -1,10 +1,7 @@
-#import <React/RCTBridgeDelegate.h>
+#import <RCTAppDelegate.h>
 #import <UIKit/UIKit.h>
-#import <UserNotifications/UNUserNotificationCenter.h>
 #import <UserNotifications/UserNotifications.h>
 
-@interface AppDelegate : UIResponder <UIApplicationDelegate, RCTBridgeDelegate, UNUserNotificationCenterDelegate>
-
-@property (nonatomic, strong) UIWindow *window;
+@interface AppDelegate: RCTAppDelegate<UNUserNotificationCenterDelegate>
```

Comment:
This is the major change in this PR. Updating syntax to align with our docs changes.